### PR TITLE
Added mass detection for IMS data overview

### PR DESCRIPTION
**Pull request by omokshyna**
Simple implementation of mass detection status for IM data overview. Shows icons for both scan and frame.
An attached video demonstrates an example of the implemented feature - https://www.loom.com/share/5643ce8ebb754df687d6121435cd901b

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java [hunk 1]
```diff
@@ -166,6 +174,22 @@ public IMSRawDataOverviewPane(final double frameNoiseLevel, final double mobilit
     unitFormat = MZmineCore.getConfiguration().getUnitFormat();
     setCenter(chartPanel);
 
+    massDetectionPane = new GridPane();
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java:177]: can you add some padding to the grid pane? like 5 on each side?

File: src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java [hunk 2]
```diff
@@ -166,6 +174,22 @@ public IMSRawDataOverviewPane(final double frameNoiseLevel, final double mobilit
...
+    massDetectionScanIcon = new FontIcon();
+    Label massDetectionScanLabel = new Label("Masses detected in the selected scan");
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java:179]: "in all mobility scans" is the technically correct term

**Review by SteffenHeu — COMMENTED** [review]
Looks good, only some minor comments

File: src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java (hunk 1 above)
Comment (omokshyna) [reply src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java:178]: Added

File: src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java (hunk 2 above)
Comment (omokshyna) [reply src/main/java/io/github/mzmine/modules/visualization/rawdataoverviewims/IMSRawDataOverviewPane.java]: Thanks, I modified the label

**Pull request merged**